test/gen_coverage_reports.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
-set -ex
+set -eux
```

Comment:
please don't add 'u'. I took it out because otherwise I cannot run `gen_coverage` locally since it barfs about undefined $CIRCLECI.